harvard-cape-peninsula-university-of-technology.csl

```diff
@@ -109,8 +109,8 @@
   <citation et-al-min="3" et-al-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true">
     <layout prefix="(" suffix=")" delimiter="; ">
       <group>
```

Comment:
Facepalm! Sorry, that was a totally dyslexic moment. Of course it 'delimiter'. It works like a charm in csl.mendeley.com now. I'll make the changes here.

And yes, you are absolutely right about the delimiter being more robust, I was wondering that.